crypto.go

```diff
@@ -320,6 +320,119 @@ func (s x448Scheme) PublicKeySize() int {
 	return 56
 }
 
+///////
+// SIKE
```

Comment:
nit: s/SIKE/SIDH here and elsewhere (SIKE is key exchange, SIDH is the primitive)

Reply:
Merged before I saw this.  But I think I'm going to keep it -- the KE is Key Encapsulation, which is what we're doing here.